ddl.dic

```diff
@@ -1729,7 +1729,7 @@ save_type.contents
 ;
          Integer
 ;
-         positive or negative integer number
+         An integer. Positive and negative integers, and zero, are included.
```

Comment:
```suggestion
         An integer. Belongs to a numerical set that consists of positive integers, negative integers and the number zero.
```

Reply:
Simply proposed an alternative formulation of the same statement. Feel free to ignore if it seems too verbose.

Reply:
It does seem a little self-referential to define a thing in terms of itself.

[Wolfram|Alpha](https://www.wolframalpha.com/examples/mathematics/numbers/integers) define integers as:
> The set of integers consists of the counting numbers, their negations and zero.

Perhaps we could use:
> An integer. Integers consist of the positive and negative counting numbers, and zero.

Reply:
The term "negative counting numbers" looks a bit off to me as counting numbers by definition should not be negative. Maybe something like "`A number from the set of all integers.`" would work? We could potentially further specify what makes up the integer set, but this seems like a well established mathematical term so there is probably no need to be too verbose. The important thing in my eyes is to specify two things:
- That the allowed values include a zero.
- That the `Integer` in question refers to the mathematical concept of an integer and not to a programmers view of an integer.

Both things seem to be covered by the phrase "set of all integers". What do you think?

Reply:
That sounds good to me.

"`A number from the set of all integers.`"